src/middlewares/__tests__/createMetadataMiddleware.ts

```diff
@@ -13,28 +13,49 @@ declare global {
   // eslint-disable-next-line @typescript-eslint/no-namespace
   namespace NodeJS {
     interface Global {
-      navigator: {
-        userAgent: string;
+      navigator?: {
+        userAgent?: string;
+        // Deprecated property, but still used in React Native
+        // https://github.com/facebook/react-native/blob/8bd3edec88148d0ab1f225d2119435681fbbba33/Libraries/Core/setUpNavigator.js
+        product?: string;
```

Comment:
```suggestion
```

we don't use it, so no need to test it

Reply:
This is useful to test the React Native use case accurately. How would you do it?